Assets/Scripts/Game/UserInterfaceWindows/DaggerfallSpellMakerWindow.cs

```diff
@@ -743,6 +743,22 @@ private void BuyButton_OnMouseClick(BaseScreenComponent sender, Vector2 position
             //   Implement costs and remove gold from player and block if user does not have enough gold (message 1702)
             //   Spells will be free (both gold and spell points) during build-out of effect system.
 
+            var moneyAvailable = GameManager.Instance.PlayerEntity.GoldPieces;
```

Comment:
To get the total amount of money available, including letters of credits, use the following helper instead :)

    GameManager.Instance.PlayerEntity.GetGoldAmount()

Reply:
Thanks, refactored :)